launch/tier4_system_launch/launch/system.launch.xml

```diff
@@ -13,15 +29,15 @@
     <!-- System Monitor -->
     <group if="$(var launch_system_monitor)">
       <push-ros-namespace namespace="system_monitor"/>
-      <include file="$(find-pkg-share system_monitor)/launch/system_monitor.launch.xml">
-        <arg name="cpu_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/cpu_monitor.param.yaml"/>
-        <arg name="hdd_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/hdd_monitor.param.yaml"/>
-        <arg name="mem_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/mem_monitor.param.yaml"/>
-        <arg name="net_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/net_monitor.param.yaml"/>
-        <arg name="ntp_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/ntp_monitor.param.yaml"/>
-        <arg name="process_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/process_monitor.param.yaml"/>
-        <arg name="gpu_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/gpu_monitor.param.yaml"/>
-        <arg name="voltage_monitor_config_file" value="$(var tier4_system_launch_param_path)/system_monitor/voltage_monitor.param.yaml"/>
+      <include file="$(find-pkg-share system_monitor)/launch/system_monitor.launch.py">
```

Comment:
Why did you change here from `.xml` to `.py`?

Reply:
Thank you for all the review! (and sorry for being late)
Resolved: https://github.com/autowarefoundation/autoware.universe/pull/2540/commits/767a0854fa00eeb8e1ab43d01c8473e1e822e2ff